pom.xml

```diff
@@ -70,6 +58,25 @@
       <groupId>org.parboiled</groupId>
       <artifactId>parboiled-java</artifactId>
       <version>1.1.7</version>
+      <exclusions>
+        <!-- Jenkins core bundles ASM 4, so if this really requires ASM 5 and is used at runtime, you are SOL. -->

```

Comment:
In other words, this plugin was bundling ASM 5 (actually several copies…) which made it look like other (library) classes bundled in the plugin were linking against that, whereas in fact the baseline version of Jenkins core bundles ASM 4, which takes precedence. Current versions bundle 5.